lesson_27/lesson_27.py

```diff
@@ -0,0 +1,44 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import time
+
+
+class NovaPoshtaTracker:
```

Comment:
It is better to use fixture for driver initialization and Page Object pattern for Page description, test class for testing. It is not so good practice for mixed up all this stuff inside one python file and class.